src/objects/events.ts

```diff
@@ -25,6 +25,7 @@ type Events = {
   fontsLoaded: Fonts,
   containerCreated: Container,
   layoutUpdated: LayoutSettings,
+  layoutCreated: LayoutSettings,
```

Comment:
Previously the layout had defaults for everything so it was never "created". But now the layout settings emit a created event when the horizontal scale is initialized.